nav2_docking/opennav_docking/include/opennav_docking/utils.hpp

```diff
@@ -183,6 +185,17 @@ inline double l2Norm(const geometry_msgs::msg::Pose & a, const geometry_msgs::ms
     delta_angle * delta_angle);
 }
 
+inline DockDirection getDockDirectionFromString(const std::string & direction)
+{
+  if (direction == "forward") {
```

Comment:
You can use `toupper` in order to convert a string to all upper case. Can you convert the input to all upper and then compare against an all upper version of forward/backward? That way if users define it in any medium of caps, it'll evaluate properly?